lib/internal/event_target.js

```diff
@@ -354,6 +349,22 @@ ObjectDefineProperties(
     isTrusted: isTrustedDescriptor,
   });
 
+const staticProps = ['NONE', 'CAPTURING_PHASE', 'AT_TARGET', 'BUBBLING_PHASE'];
+
+ObjectDefineProperties(
+  Event,
+  staticProps.reduce((result, staticProp, index = 0) => {
```

Comment:
Sorry for asking another change but in cases like this, the primordial should be used, this should be changed to:

```suggestion
  ArrayPrototypeReduce(staticProps, (result, staticProp, index = 0) => {
```

You can add the import below line `4` of this file:

```
'use strict';

const {
  ArrayFrom,
  ArrayPrototypeReduce
```

After this, I think it will be very good to be released, thanks @BenzeneAlcohol  for the patience.

Reply:
Done.

It is fine, just small changes that I can do to maintain the quality.